tools/vsimporter/xib2nib/UIStoryboardSegue.cpp

```diff
@@ -31,10 +31,13 @@ void UIStoryboardSegue::InitFromStory(XIBObject* obj) {
     ObjectConverter::InitFromStory(obj);
     _destination = getAttrAndHandle("destination");
     if (_destination) {
-        char szDestination[255];
-        sprintf(szDestination, "UIViewController-%s", _destination);
-        _destination = strdup(szDestination);
+        // IMPORTANT: Commenting out the next few lines because it this blocks segue handling when NIB filenames are prefixed with "UIViewController-".
+        //            This prefix is prepended generated under the IBTool (compatibility purposes) but it's not required by WinObjC-generated usage.
+        // char szDestination[255];
+        // sprintf(szDestination, "UIViewController-%s", _destination);
+        //_destination = strdup(szDestination);
```

Comment:
DO we need a todo with  tracking issue for something or should we just delete the commented out code? 

Commented out code pollutes code bases as future maintainers rarely have full context (even with comments like this) to understand how commented out code may / may not fit in with potentially vastly diverged compiling code. Rarely does the commented code add value as git history and other tools can show what used to be present and any "future work" should be tracked with issues so that it will definitely be addressed in a follow up review.